WindowsGSM/MainWindow.xaml.cs

```diff
@@ -2656,9 +2664,16 @@ private async Task EndAllRunningProcess(string serverId)
             });
         }
 
-        private void SetServerStatus(ServerTable server, string status)
+        private void SetServerStatus(ServerTable server, string status, string pid)
```

Comment:
how about this? 👍 
```cs
private void SetServerStatus(ServerTable server, string status, string pid = null)
```

Reply:
Yea, that would be much better! Didn't think about that.. At all.